lib/spoom/deadcode/plugins/minitest.rb

```diff
@@ -7,21 +7,39 @@ module Plugins
       class Minitest < Base
         extend T::Sig
 
-        ignore_classes_named(/Test$/)
-
-        ignore_methods_named(
-          "after_all",
-          "around",
-          "around_all",
-          "before_all",
-          "setup",
-          "teardown",
+        ignore_classes_inheriting_from("Minitest::Test")
+
+        MINITEST_METHODS = T.let(
+          Set.new([
+            "after_all",
+            "around",
+            "around_all",
+            "before_all",
+            "setup",
+            "teardown",
+          ]),
+          T::Set[String],
         )
 
         sig { override.params(definition: Model::Method).void }
         def on_define_method(definition)
-          file = definition.location.file
-          @index.ignore(definition) if file.match?(%r{test/.*test\.rb$}) && definition.name.match?(/^test_/)
+          return unless definition.name =~ /^test_/ || MINITEST_METHODS.include?(definition.name)
```

Comment:
regex is overkill here

```suggestion
          return unless definition.name.start_with?("test_") || MINITEST_METHODS.include?(definition.name)
```